ui/app/AppLayouts/Profile/views/CommunitiesView.qml

```diff
@@ -53,35 +52,32 @@ SettingsContentBase {
 
             CommunitiesListPanel {
                 width: parent.width
-                model: root.profileSectionStore.communitiesList
+                model: root.communitiesList
 
                 onLeaveCommunityClicked: {
-                    root.profileSectionStore.communitiesProfileModule.leaveCommunity(communityId)
+                    root.store.leaveCommunity(communityId)
                 }
 
                 onSetCommunityMutedClicked: {
-                    root.profileSectionStore.communitiesProfileModule.setCommunityMuted(communityId, muted)
+                    root.store.setCommunityMuted(communityId, muted)
                 }
 
                 onSetActiveCommunityClicked: {
-                    rootStore.setActiveCommunity(communityId)
+                    root.store.setActiveCommunity(communityId)
                 }
 
                 onInviteFriends: {
-                    Global.openInviteFriendsToCommunityPopup(communityData,
-                                                             root.profileSectionStore.communitiesProfileModule)
+                    Global.openInviteFriendsToCommunityPopup(communityData, root.store)
                 }
             }
-
         } // Column
     } // Item
 
     property Component importCommunitiesPopupComponent: ImportCommunityPopup {
         anchors.centerIn: parent
-        store: root.profileSectionStore
+        store: root.store
```

Comment:
Before it was `ProfileSectionStore` but now it's a `GlobalStore`. Are you sure about it?